client/src/components/ElementActions/tests/UnpublishAction-test.js

```diff
@@ -1,95 +1,88 @@
 /* eslint-disable import/no-extraneous-dependencies */
-/* global jest, describe, it, expect, window */
+/* global jest, test, describe, it, expect, window */
 
 import React from 'react';
 import { Component as UnpublishAction } from '../UnpublishAction';
-import Enzyme, { mount } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import { fireEvent, render } from '@testing-library/react';
 
-Enzyme.configure({ adapter: new Adapter() });
+window.jQuery = {
+  noticeAdd: jest.fn()
+};
 
-describe('UnpublishAction', () => {
-  let wrapper = null;
-  const mockMutation = jest.fn(() => new Promise((resolve) => { resolve(); }));
-  const WrappedComponent = (props) => <div>{props.children}</div>;
-  const ActionComponent = UnpublishAction(WrappedComponent);
-  const jQuery = jest.fn();
-  window.jQuery = jQuery;
-
-  beforeEach(() => {
-    wrapper = mount(
-      <ActionComponent
-        title="My abstract action"
-        element={{
-          id: 123,
-          isPublished: true,
-          blockSchema: { type: 'Test' },
-        }}
-        type={{ title: 'Some block' }}
-        actions={{ handleUnpublishBlock: mockMutation }}
-        toggle={false}
-      />
-    );
-
-    jQuery.noticeAdd = jest.fn();
-  });
-
-  it('renders the wrapped component', () => {
-    expect(wrapper.children().first().type()).toEqual(WrappedComponent);
-  });
-
-  it('renders a button', () => {
-    expect(wrapper.find('button').length).toBe(1);
-  });
-
-  it('renders the title and class', () => {
-    expect(wrapper.find('button').text()).toContain('Unpublish');
-    expect(wrapper.find('button').hasClass('element-editor__actions-unpublish')).toBe(true);
-  });
+function makeProps(obj = {}) {
+  return {
+    title: 'My abstract action',
```

Comment:
```suggestion
    title: 'My unpublish action',
```
I got confused as to what this was testing for a second because of this. 😅

Reply:
Updated